### DISCUSSION TITLE
How to create netty SslContext from java SslContext

### DISCUSSION THREAD

**Discussion by ShlomiRex** [netty/netty#12859] [General]
Title

I'm so confused. I have a single truststore file (with extension 'jks'). I have nothing else (and password to the file).
How can I convert `javax.net.ssl.SSLContext` to `io.netty.handler.ssl.SSLContext` ?

**Comment by chrisvest** [comment]
Only the `JdkSslContext` implementation let you do that, with the `context()` method.
